### ISSUE TITLE
TabView: Screen reader does not provide any confirmation after opening/closing new tab

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#3746]
**Describe the bug**
Opening and closing a (new) tab when a screen reader is active should notify the user with a confirmation message.

**Steps to reproduce the bug**
1. Go to TabView page in XAML Controls Gallery
2. Enable your screen reader (I used Narrator and NVDA)
3. Use your keyboard or mouse to navigate open or close a (new) tab

**Expected behavior**
In Chromium Edge, opening a new tab has Narrator/NVDA say "opening new tab" and closing a tab has Narrator/NVDA say "closing tab". This behavior should be consistent with the TabView.

**Screenshots**
Not a visual bug

**Version Info**
Using XAML Controls Gallery v1.2.16.0

NuGet package version: ?

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
| Yes | No? |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            |  |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**
I think this is a TabView bug, not a XAML Controls Gallery bug.
https://github.com/microsoft/terminal/issues/8464 was filed on Windows Terminal during accessibility testing.

**Comment by chingucoding** [comment]
@YuliKl @stmoy Do you think that this is already covered by #1883? Otherwise, what do you think would be the best way to fix this (especially given that #3469 still seems to be an issue)?

**Comment by carlos-zamora** [comment]
This is still an issue. Unless y'all want to dup it to #1883.

**Comment by DmitriyKomin** [comment]
@carlos-zamora, thank you for reporting the issue. Unfortunately we are not currently investigating WinUI2 issues that are not business or security critical (more on the policy [here](https://github.com/microsoft/microsoft-ui-xaml/blob/main/docs/contribution_handling.md#winui-2)).
